tests/integration/test_helpers.go

```diff
@@ -169,15 +154,11 @@ func InitTestSuite() {
 
 func SetupCommonResources(namePrefix string) {
 	namespace = fmt.Sprintf("%s-%s", namePrefix, generateRandomString(6))
-	randomSuffix6 := generateRandomString(6)
-	oauthSecretName := fmt.Sprintf("%s-secret-%s", namePrefix, randomSuffix6)
-	oauthClientName := fmt.Sprintf("%s-client-%s", namePrefix, randomSuffix6)
 	log.Printf("Using namespace: %s\n", namespace)
-	log.Printf("Using OAuth2Client with name: %s, secretName: %s\n", oauthClientName, oauthSecretName)
 	oauth2Cfg = &clientcredentials.Config{
 		ClientID:     conf.ClientID,
 		ClientSecret: conf.ClientSecret,
-		TokenURL:     fmt.Sprintf("%s/oauth2/token", conf.IasAddr),
+		TokenURL:     fmt.Sprintf("%s/oauth2/token", conf.IssuerUrl),
 		Scopes:       []string{"read"},
```

Comment:
Seems like most of those parameters aren't used right now

Reply:
These are for obtaining a token within the test scenario (see https://github.com/kyma-project/api-gateway/blob/60ef392d94ac6b7f451fedd546138fd5775457c7/tests/integration/pkg/jwt/auth.go#L29). I did remove all other unused. Hopefully got them all.

Reply:
Is `Scopes` really needed?

Reply:
Ah, you meant scopes, perhaps will work without it. Will test.